components/event-sources/reconciler/httpsource/reconcile_test.go

```diff
@@ -484,3 +659,14 @@ func newServiceNotReady() *servingv1alpha1.Service {
 	}})
 	return svc
 }
+
+// Ready: False without a Revision
+func newServiceNotReadyWithRevision() *servingv1alpha1.Service {
```

Comment:
Mismatch:
- "without a Revision"
- "WithRevision()"

Reply:
Fixed the comment